src/pages/Signin.tsx

```diff
@@ -0,0 +1,10 @@
+import { Login } from "./Login";
+
+export function Signin() {
+  return (
+    <div className="signin">
+      {/* navbar will be placed there */}
+      <Login />
+    </div>
+  );
+}
```

Comment:
The same navbar will be on all pages, so it should be implemented using `Outlet`, not here.